mirage/factories/owaspmobile2024.ts

```diff
@@ -0,0 +1,11 @@
+import { Factory } from 'miragejs';
+import { faker } from '@faker-js/faker';
+
+export default Factory.extend({
```

Comment:
Can the factory object be put in a variable? Similar to how the vulnerability factory is done. This is for the cypress tests.